tests/program/test_validate.py

```diff
@@ -219,8 +219,9 @@ def test_program_validate_zipped_resources_979():
     assert result.stdout.count("valid: ogd10_energieforschungstatistik_ch.csv")
     assert result.stdout.count("valid: ogd10_catalogs.zip => finanzquellen.csv")
     assert result.stdout.count("invalid: ogd10_catalogs.zip => capital-invalid.csv")
+    assert result.stdout.count("Schema is not valid")
     assert result.stdout.count(
-        "Schema is not valid: Schemas with duplicate field names are not supported"
+        "Schemas with duplicate field\n                error    names are not supported\n\n"
```

Comment:
Can we use a multiline string here?

Reply:
I had tried that but it did not work because the stdout in cli stores output string as. 
```
...
Schemas with duplicate field\n                error    names are not supported\n\n
...
```
However this works:
```
    assert result.stdout.count(
        """row    field    code     message
-----  -------  -------  -------------------------------------------------
                schema-  Schema is not valid: Schemas with duplicate field
                error    names are not supported"""
    )
```